pandera/api/polars/model.py

```diff
@@ -94,8 +97,8 @@ def _build_columns(  # pylint:disable=too-many-locals
                     f"{annotation.origin.__name__}"
                 )
                 msg = (
-                    " Series[TYPE] annotations are not supported for polars. "
-                    "Use the bare TYPE directly"
+                    " pandera.typing.Series[TYPE] annotations are not supported for polars. "
+                    "Use the pandera.typing.polars.Series[TYPE] annotation or the bare TYPE directly"
```

Comment:
let's delete this part, now that polars backend now supports `Series` generic type.